pkgs/by-name/bl/bleep/package.nix

```diff
@@ -0,0 +1,89 @@
+{
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  installShellFiles,
+  makeWrapper,
+  lib,
+  zlib,
+}:
+let
+  platform =
+    {
+      x86_64-linux = "x86_64-pc-linux";
+      x86_64-darwin = "x86_64-apple-darwin";
+      aarch64-darwin = "arm64-apple-darwin";
+    }
+    ."${stdenv.system}";
+
+  hash =
+    {
+      x86_64-linux = "sha256-ttfmTMSK6u2zX87CUYjibr675Xzn4/byWub67e5Ir+I=";
+      x86_64-darwin = "sha256-bh1yQ7KZ+cwVHTLG+gpYDLPKG2VUcrz88Sqd2tRiKfM=";
+      aarch64-darwin = "sha256-IV2JPXibv80Odb+1Zkw1YUoutHsi5YaIjyNpAOIUlr4=";
+    }
+    ."${stdenv.system}";
+in
+stdenv.mkDerivation rec {
+  pname = "bleep";
+  version = "0.0.7";
+
+  src = fetchurl {
+    url = "https://github.com/oyvindberg/bleep/releases/download/v${version}/bleep-${platform}.tar.gz";
+    sha256 = hash;
+  };
```

Comment:
Why aren't you using `fetchFromGitHub` and building from source? And if you're going to use prebuilt binaries, you should use `stdenvNoCC`.

Reply:
I changed to NoCC. Bleep builds bleep itself (including graalvm-native-image packaging) and I have no idea on how I would bootstrap this.

Reply:
> Bleep builds bleep itself

That seems... flawed.